backend/auth/auth.go

```diff
@@ -0,0 +1,75 @@
+package auth
+
+import (
+	"fmt"
+	"github.com/MicahParks/keyfunc"
+	"github.com/ZPI-2024-25/KubernetesAccessManager/models"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/joho/godotenv"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+var jwks *keyfunc.JWKS
+
+func init() {
+	err := godotenv.Load()
```

Comment:
This is intentional - the error itself does not concern us, because in the production case the variables will be taken from environment variables, not from the .env file. Such a notation allows not to initialize the error itself as an empty value to assign it only at a later stage, and when it is created at the function whose error is handled then there are problems with assigning jwks. If it strongly “stings the eye” I can make logging of information that variables were not loaded from the file.

Reply:
Nah